interface/src/app/sample/sampleMocks.spec.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.mocks.sample', [])
+
+.factory('SampleMocks', [function() {
```

Comment:
Good thinking. I will change it to a `.constant`.

- [ ] change `SampleMocks` from a `.factory` to a `.constant`